lib/src/css.dart

```diff
@@ -12,6 +12,15 @@ class CSS {
   // The bootstrap css file
   final String cssFilePath = '/packages/bootstrap/css/bootstrap.css';
 
+  final String cssHeader =
+      'https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css';
+
+  final String theme =
+      'https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap-theme.min.css';
+
+  final String jsScript =
```

Comment:
Keerti, fyi, I don't think any of the bits of bootstrap that dartdoc is using requires the JS file, just the main css file and the theme one.